pdns/recursordist/rec-main.cc

```diff
@@ -2296,6 +2296,8 @@ static int serviceMain(Logr::log_t log)
   g_paddingTag = ::arg().asNum("edns-padding-tag");
   g_paddingOutgoing = ::arg().mustDo("edns-padding-out");
 
+  setAuthCookies(::arg().mustDo("outgoing-cookies"));
```

Comment:
to go with my 'enable' suggestion, maybe 'setOutgoingCookies' is already an improvement. Bonus points, it matches the config item